unlock-app/src/components/creator/lock/LockIconBar.js

```diff
@@ -38,7 +41,21 @@ export function LockIconBar({
       <IconBarContainer>
         <IconBar>
           <Buttons.Withdraw as="button" lock={lock} />
-          <Buttons.Edit as="button" />
+          {editing && (
+            <Buttons.Save
+              as="button"
+              lock={lock}
+              toggleEditing={toggleEditing}
```

Comment:
I wonder if the `Buttons.Save` button should not have a `onClick` and `onCancel` properties which would be triggered respectively for each of `updateLockPrice` and `toggleEditing` ?

Reply:
Thats much better naming, will change now

Reply:
so how do we want to trigger `onCancel`? In my head, onCancel is part of whatever editing modal is higher up, like in CreatorLock/LockRow. 

So maybe when you hit escape? Or should we create another component to cancel the edit? Or should that not be solved in this particular PR but made a different issue?

Reply:
nvm, dumb mistake. will add onCancel